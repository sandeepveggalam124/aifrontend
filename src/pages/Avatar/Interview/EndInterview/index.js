import React from 'react'
import { useLocation } from 'react-router-dom';
import ImageComponent from '../../Landing/AvatarImage/index';
import EndCard from '../EndCard';

const EndNote = () => {
  
  const avatarSrc='/assests/images/avatar.svg'
  const avatarAlt='Avatar Image'

  return (
    <div className="row custom-div text-white">
      <div className="col col-lg-1 bg-last-image"></div>

        <div className="col col-lg-10 bg-black d-flex flex-column align-item-stretch position-relative p-0">
        <ImageComponent src={avatarSrc}  alt={avatarAlt}/>
        {/* </div> */}
      
      {/* <div className="col col-lg-4 bg-black d-flex flex-column align-items-left justify-content-center"> */}
      <div className='position-absolute m-4' style={{right:'10%',bottom:'40%'}}>        
        <EndCard/>
      </div>

    </div>
      <div className="col col-lg-1 bg-last-image"></div>
    </div>
  )
}

export default EndNote