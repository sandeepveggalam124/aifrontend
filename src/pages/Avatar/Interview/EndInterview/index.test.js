import { MemoryRouter as Router } from "react-router-dom";
import {render} from "@testing-library/react";
import EndInterview from "./index";


it('renders correctly',()=>{
    const {container}=render(
        <Router>
            <EndInterview/>
        </Router>
    )
    expect(container).toMatchSnapshot()
})