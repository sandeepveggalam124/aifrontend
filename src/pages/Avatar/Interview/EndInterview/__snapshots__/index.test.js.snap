// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="row custom-div text-white"
  >
    <div
      class="col col-lg-1 bg-last-image"
    />
    <div
      class="col col-lg-10 bg-black d-flex flex-column align-item-stretch position-relative p-0"
    >
      <div
        class="embed-container p-0"
      >
        <img
          alt="Avatar Image"
          loading="lazy"
          src="/assests/images/avatar.svg"
        />
      </div>
      <div
        class="position-absolute m-4"
        style="right: 10%; bottom: 40%;"
      >
        <div
          class="card bg-black"
          style="width: 25rem;"
        >
          <div
            class="card-body d-flex flex-column justify-content-center align-items-center text-center"
          >
            <p
              class="card-text"
            >
              Thank you for completing your interview with us. We appreciate your time and interest in our company. We will be in touch with you regarding the next steps in our hiring process. Have a great day!
            </p>
            <button
              class="btn btn-light fs-6 fw-medium"
            >
              <a
                class="text-decoration-none text-dark"
                href="/"
              >
                Ok
              </a>
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="col col-lg-1 bg-last-image"
    />
  </div>
</div>
`;
